src/PrestaShopBundle/Resources/config/services/bundle/twig.yml

```diff
@@ -69,3 +69,13 @@ services:
             - '@prestashop.core.util.helper_card.documentation_link_provider'
         tags:
             - { name: twig.extension }
+
+    prestashop.twig.extension.context_iso_code_provider_extension:
+      class: PrestaShopBundle\Twig\ContextIsoCodeProviderExtension
+      arguments:
+        - "@=service('prestashop.adapter.legacy.context').getContext().language.iso_code"
```

Comment:
Careful with this one, if language is empty this simple definition will crash the whole framework. It's especially true and sensitive with Twig extension because if they fail you can't even use `./bin/console` any more
It happened with `prestashop.core.util.helper_card.documentation_link_provider`, you can use the same kind of "safety":

```yaml
  prestashop.core.util.helper_card.documentation_link_provider:
    class: 'PrestaShop\PrestaShop\Core\Util\HelperCard\DocumentationLinkProvider'
    arguments:
      - '@=service("prestashop.adapter.legacy.context").getContext().language ? service("prestashop.adapter.legacy.context").getContext().language.iso_code : "en-US"'
```

Reply:
Also check if basic commands like `./bin/console debug:container` or `./bin/console debug:router` work correctly

Reply:
Actually tested it, `debug:container` and `debug:router` are fine, but `./bin/console` alone has an error:

```
08:17:23 ERROR     [console] An error occurred while using the console. Message: "Notice: Trying to get property 'iso_code' of non-object" ["exception" => Symfony\Component\Debug\Exception\ContextErrorException { …},"message" => "Notice: Trying to get property 'iso_code' of non-object"]

In appDevDebugProjectContainer.php line 2202:

  Notice: Trying to get property 'iso_code' of non-object```

Reply:
Thank you - iplemented your suggested fix and right now `./bin/console` works just fine

Reply:
Great thanks @tomas862